lonestar/analytics/cpu/sssp/sssp_cli.cpp

```diff
@@ -140,7 +140,7 @@ main(int argc, char** argv) {
 
   std::cout << "Running " << AlgorithmName(algo) << " algorithm\n";
```

Comment:
Line47: Can you please change `clEnumVal` to `clEnumValN` for `algo` options? Also, in sssp/CMakeLists.txt add` --algo` explicitly. 
I think the same goes for bfs_cli.cpp.

Reply:
Done!